cruise-control-client/cruisecontrolclient/client/cccli.py

```diff
@@ -0,0 +1,353 @@
+        for alias in endpoint_aliases:
+            dest_to_Endpoint[alias] = endpoint
+    for parameter in endpoint.available_Parameters:
+        available_parameter_set.add(parameter)
+
+# A mapping from the names of the argparse parameters to their cruise-control parameter name
+#
+# Like:
+# {'allow_capacity_estimation': 'allow_capacity_estimation',
+#  'brokers': 'brokerid',
+#  'clearmetrics': 'clearmetrics',
+#  'client_id': 'client_ids',
+#  'concurrency': 'concurrent_partition_movements_per_broker',
+#  'data_from': 'data_from',
+#  'disable_self_healing_for': 'disable_self_healing_for',
+#  'dry_run': 'dryRun',
+#  'enable_self_healing_for': 'enable_self_healing_for',
+#  'end_timestamp': 'end',
+#  'endpoint': 'endpoints',
+#  'exclude_follower_demotion': 'exclude_follower_demotion',
+#  'exclude_recently_demoted_brokers': 'exclude_recently_demoted_brokers',
+#  'exclude_recently_removed_brokers': 'exclude_recently_removed_brokers',
+#  'excluded_topics': 'excluded_topics',
+#  'goals': 'goals',
+#  'ignore_proposal_cache': 'ignore_proposal_cache',
+#  'leader_concurrency': 'concurrent_leader_movements',
+#  'max_load': 'max_load',
+#  'min_valid_partition_ratio': 'min_valid_partition_ratio',
+#  'number_of_entries_to_show': 'entries',
+#  'partition': 'partition',
+#  'reason': 'reason',
+#  'resource': 'resource',
+#  'skip_hard_goal_check': 'skip_hard_goal_check',
+#  'skip_urp_demotion': 'skip_urp_demotion',
+#  'start_timestamp': 'start',
+#  'strategies': 'replica_movement_strategies',
+#  'substate': 'substates',
+#  'super_verbose': 'super_verbose',
+#  'text': 'json',
+#  'throttle_removed_broker': 'throttle_removed_broker',
+#  'timestamp': 'time',
+#  'topic': 'topic',
+#  'types': 'types',
+#  'use_ready_default_goals': 'use_ready_default_goals',
+#  'user_task_ids': 'user_task_ids',
+#  'verbose': 'verbose'}
+flag_to_parameter_name: Dict[str, str] = {}
+for parameter in available_parameter_set:
+    argparse_parameter_name = None
+    # argparse names this parameter's flag after the first string in 'args'
+    for possible_argparse_name in parameter.argparse_properties['args']:
+        # argparse chooses flag names only from the --flags
+        if not possible_argparse_name.startswith('-'):
+            argparse_parameter_name = possible_argparse_name.replace('-', '_')
+        elif not possible_argparse_name.startswith('--'):
+            continue
+        else:
+            argparse_parameter_name = possible_argparse_name.lstrip('-').replace('-', '_')
+            break
+    if argparse_parameter_name and argparse_parameter_name in flag_to_parameter_name:
+        raise ValueError(f"Colliding parameter flags: {argparse_parameter_name}")
+    else:
+        flag_to_parameter_name[argparse_parameter_name] = parameter.name
+
+
+def query_cruise_control(args):
+    """
+    Handles asking cruise-control for a requests.Response object, and pretty-
```

Comment:
Nit (same occurrences in 18 locations): `requests.Response ` -> `requests. Response ` (missing space after dot)

Reply:
Not missing space, this refers to the `Response` object of the `requests` module